legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -393,7 +405,11 @@ def party_revision_json(transaction_id, party_revision) -> dict:
             if party_revision.delivery_address:
                 member['mailingAddress'] = member['deliveryAddress']
 
-        member['id'] = party_revision.id
+        if is_ia_or_after:
+            member['officer']['id'] = party_revision.id
```

Comment:
Convert these ids to string type. ids are there in almost all objects except addresses I think. Whereever id is being returned currently add convert that to string

Reply:
ok